src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -637,22 +638,39 @@ TEST_F(TransportManagerImplTest, UpdateDeviceList_AddNewDevice) {
   device_list_.pop_back();
 }
 
+struct DeviceInfoComparator {
+    const transport_manager::DeviceInfo& val_;
```

Comment:
@LuxoftAKutsan should it be named as **kVal_**?
and maybe it better to name it something like **kDeviceInfoVal_**?